mos_tests/murano/actions.py

```diff
@@ -0,0 +1,210 @@
+    def get_action_id(self, environment, name, service):
+        env_data = environment.to_dict()
+        a_dict = env_data['services'][service]['?']['_actions']
+        for action_id, action in a_dict.items():
+            if action['name'] == name:
+                return action_id
+
+    def run_action(self, environment, action_id):
+        self.os_conn.murano.actions.call(environment.id, action_id)
+        return self.wait_for_environment_deploy(environment)
+
+    def status_check(self, environment, configurations, kubernetes=False,
+                     negative=False):
+        for configuration in configurations:
+            if kubernetes:
+                service_name = configuration[0]
+                inst_name = configuration[1]
+                ports = configuration[2:]
+                ip = self.get_k8s_ip_by_instance_name(environment, inst_name,
+                                                      service_name)
+                if ip and ports and negative:
+                    for port in ports:
+                        assert self.check_port_access(ip, port, negative)
+                        assert self.check_k8s_deployment(ip, port, negative)
+                elif ip and ports:
+                    for port in ports:
+                        assert self.check_port_access(ip, port)
+                        assert self.check_k8s_deployment(ip, port)
+                else:
+                    assert 0, "Instance {} doesn't have floating IP"\
+                        .format(inst_name)
+            else:
+                inst_name = configuration[0]
+                ports = configuration[1:]
+                ip = self.get_ip_by_instance_name(environment, inst_name)
+                if ip and ports:
+                    for port in ports:
+                        assert self.check_port_access(ip, port)
+                else:
+                    assert 0, "Instance {} doesn't have floating IP"\
+                        .format(inst_name)
+
+    def check_port_access(self, ip, port, negative=False):
+        result = 1
+        start_time = time.time()
+        while time.time() - start_time < 600:
+            sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+            result = sock.connect_ex((str(ip), port))
+            sock.close()
+
+            if result == 0 or negative:
+                break
+            time.sleep(5)
+        if negative:
+            assert result != 0, '{} port is opened on instance'.format(port)
+        else:
+            assert result == 0, '{} port is closed on instance'.format(port)
+        return True
+
+    def check_k8s_deployment(self, ip, port, timeout=3600, negative=False):
+        start_time = time.time()
+        while time.time() - start_time < timeout:
+            try:
+                self.verify_connection(ip, port, negative)
+                return True
+            except RuntimeError:
+                time.sleep(10)
+        assert 0, 'Containers are not ready'
```

Comment:
I don't like this string. if we want to raise exception here let's use 'raise' then.